rust/proto_generator/build.rs

```diff
@@ -15,7 +15,9 @@ fn main() {
         .out_dir(out_dir)
         .compile(
             &[
+                format!("{proto_dir}/permissions.proto"),
```

Comment:
You need to go and add these files to the generator scripts for each language - we hardcode the names of the files.

Reply:
good catch Ramya, thank you!